src/kiota/kiota.csproj

```diff
@@ -26,6 +26,10 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\Microsoft.OpenApi.snk</AssemblyOriginatorKeyFile>
     <Nullable>enable</Nullable>
+
+    <!-- DEBUG Source Generators -->
+    <!-- <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles> -->
```

Comment:
can you provide some context on why are those commented?

Reply:
Those are useful only when debugging the Source Generator itself

Reply:
removed